Formula/p/parallel-hashmap.rb

```diff
@@ -1,9 +1,11 @@
 class ParallelHashmap < Formula
   desc "Family of header-only, fast, memory-friendly C++ hashmap and btree containers"
   homepage "https://greg7mdp.github.io/parallel-hashmap/"
-  url "https://github.com/greg7mdp/parallel-hashmap/archive/refs/tags/1.37.tar.gz"
-  sha256 "2ac652be0552fcb53a1163c08c1f28f29f0756594fcc587eebb4d8b363153709"
+  url "https://github.com/greg7mdp/parallel-hashmap/archive/refs/tags/v1.3.12.tar.gz"
+  sha256 "0cc203144321924cfbfcc401f42d8204c0dd24e2760c7a1c091baa16d9777c08"
   license "Apache-2.0"
+  # upsteam versioning scheme changed between v1.37 and v1.3.8
+  version_scheme 1
```

Comment:
Do you have confirmation from upstream that this change was intentional?

Reply:
The [release notes for 1.3.8](https://github.com/greg7mdp/parallel-hashmap/releases/tag/v1.3.8) contain:
> - Update version numbers to semantic versioning.